python/phonenumbers/__init__.pyi

```diff
@@ -0,0 +1,82 @@
+from .asyoutypeformatter import AsYouTypeFormatter as AsYouTypeFormatter
```

Comment:
It is required, although looking at the docs it seems it might also be possible to put in `__all__` -- though in other stub files I've seen it like this and thought it was the canonical approach

https://mypy.readthedocs.io/en/stable/config_file.html#confval-implicit_reexport (note the last line that implicit re-export is disabled for stubs -- the docs could be written more clearly here on the inverse of explicit re-exports)